pkg/helpers/contexts/heartbeat.go

```diff
@@ -30,12 +29,12 @@ type Actuator struct {
 
 // WithHeartBeat with return a context that is automatically cancelled if the HeartBeat function
 // from the returned Actuator is not invoked periodically before the passed lifetime expires.
-func WithHeartBeat(parent context.Context, lifeTime time.Duration) (context.Context, Actuator) {
```

Comment:
Since we renamed this parameter, I think it would be great if we also rename it in all the callers (for consistency). WDYT?